build_functions/CMakeLists.txt

```diff
@@ -266,3 +266,36 @@ function(add_reals_check_target)
         )
     endif()
 endfunction()
+
+# signature: install_library_with_prefix(<theTarget> <includePrefix> [ <public_header>... ]])
+# Creates an install target for library with a subdirectory and optional public header files.
+function(install_library_with_prefix theTarget includePrefix)
+    
+    if(ARGN)
+        set_target_properties(${theTarget} PROPERTIES PUBLIC_HEADER "${ARGN}")
+    endif()
+
+    # Set CMAKE_INSTALL_* if not defined
+    include(GNUInstallDirs)
+
+    if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+        set(CMAKE_INSTALL_LIBDIR "lib")
+    endif()
+
+    # Install elastic_log_rpc_client
```

Comment:
>elastic_log_rpc_client [](http://example.com/codeflow?start=14&length=22)

This is not really true now that this is a generic function.

Reply:
Urgh!